test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/Infrastructure/DthTestServer.cs

```diff
@@ -19,12 +19,12 @@ public class DthTestServer : IDisposable
         private const string DthName = "Microsoft.Dnx.DesignTimeHost";
         private Process _process;
 
-        public static DthTestServer Create(string runtimeHomePath, string projectPath, TimeSpan timeout)
+        public static DthTestServer Create(DnxSdk sdk, string projectPath, TimeSpan timeout)
```

Comment:
Should we either add `Dth` as a property on `DnxSdk` or maybe make this an extension method? I'd rather keep using `DnxSdk` as the hub for all things related to a particular SDK version.